test/SQLBuilderTest.php

```diff
@@ -3,15 +3,20 @@
 use ActiveRecord\Exception\ActiveRecordException;
 use ActiveRecord\SQLBuilder;
 use ActiveRecord\Table;
+use ActiveRecord\WhereClause;
 use test\models\Author;
 
```

Comment:
Rename to class WhereTest

Reply:
I dunno, man, the tests are a big crazy mish-mash and sometimes it's hard to isolate stuff. If you have ideas about reorganizing them feel free to open a PR, and I will stamp away.